slasher/rpc/server.go

```diff
@@ -91,5 +91,37 @@ func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Indexed
 // IsSlashableBlock returns an proposer slashing if the block submitted
 // is a double proposal.
 func (ss *Server) IsSlashableBlock(ctx context.Context, req *ethpb.SignedBeaconBlockHeader) (*slashpb.ProposerSlashingResponse, error) {
-	return nil, errors.New("unimplemented")
+	ctx, span := trace.StartSpan(ctx, "detection.IsSlashableBlock")
+	defer span.End()
+
+	gvr, err := ss.beaconClient.GenesisValidatorsRoot(ctx)
```

Comment:
This value should only be fetched once, any chance we can retrieve it somewhere else or cache future calls?

Reply:
thanks, added